.gitignore

```diff
@@ -8,3 +8,7 @@ mitocheck_control_features
 tmp/
 # PyBaSiC
 idrstream/PyBaSiC/
+# Juypter testing notebook
```

Comment:
I think when you are completely done testing it would be a good idea to move this notebook to a different location (and  delete these lines from the gitignore). This way, users know what files they should expect to see on their computer when the example notebook is run.

Ex:
When someone sets up `IDR_stream` and runs an example notebook they would expect to see  `idrstream/PyBaSic/` and `tmp/` directories on their computer, but these would be too large and necessary to upload to Github. However, this gitignore shows that they should expect to have those files on their computer.

Reply:
:+1: